package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "timed",
-  "version": "1.3.2",
+  "version": "1.3.4",
```

Comment:
Jumping from `v1.3.2` to `v1.3.4` because the change in this file didn't happen at the time we released `v1.3.3`.